client/src/components/Post.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react"
+import styled from "styled-components"
+
+const PostContainer = styled.div`
+    box-shadow: 
+        -1px -2px 5px rgba(0, 0, 0, 0.05),
+        2px 3px 5px rgba(0, 0, 0, 0.15);
+    margin: 0.5rem 0;
+    padding: 0.5rem;
+    width: 75%;
+`
+
+export default class Post extends Component {
+    state = {
```

Comment:
Considering that this pull request is just for styling the web app, I think this voting logic should be moved to another pull request.